File: Q5.cpp
#include <iostream>

using namespace std;

int main()
{
    string nome;
    int vezes = 1;
    cout << "Digite o seu nome: ";
    getline(cin, nome);
    for (int contador = 0; contador < 10; contador ++, vezes ++)
    {
        cout << vezes << "ª Vez: " << nome << "." << endl;
    }
    return 0;
}

File: Q5.cs
using System;

namespace Q5
{
    class Program
    {
        static void Main(string[] args)
        {
            string nome;
            byte vezes = 1;
            Console.Write("Digite o seu nome: ");
            nome = Console.ReadLine();
            for (byte contador = 0; contador < 10; contador ++, vezes ++)
            {
                Console.WriteLine("{0}ª Vez: {1}.", vezes, nome);
            }
        }
    }
}